api/all.yaml

```diff
@@ -1490,6 +1832,206 @@ components:
+        FAILED_REGISTRATION status indicates that the client has failed to write all model artifacts to external storage.
+        READY status indicates that the client has successfully written all model artifacts to external storage.
+      type: string
+      enum:
+        - MODEL_VERSION_STATUS_UNKNOWN
+        - PENDING_REGISTRATION
+        - FAILED_REGISTRATION
+        - READY
+    CreateRegisteredModel:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        storage_location:
+          description: Storage root URL for registered model (for **MANAGED**, **EXTERNAL** regisetered models)
+          type: string
+        comment:
+          description: User-provided free-form text description.
+          type: string
+      required:
+        - model_name
+        - catalog_name
+        - schema_name
+    UpdateRegisteredModel:
+      type: object
+      properties:
+        comment:
+          description: User-provided free-form text description.
+          type: string
+        new_name:
+          description: New name for the model.
+          type: string
+    ListRegisteredModelsResponse:
+      type: object
+      properties:
+        model_versions:
+          description: An array of model information objects.
+          type: array
+          items:
+            "$ref": "#/components/schemas/RegisteredModelInfo"
+        next_page_token:
+          type: string
+          description: |
+            Opaque token to retrieve the next page of results. Absent if there are no more pages.
+            __page_token__ should be set to this value for the next request (for the next page of results).
+    CreateModelVersion:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        source:
+          description: URI indicating the location of the source model artifacts.
+          type: string
+        run_id:
+          description: The run id used by the ML package that generated the model.
+          type: string
+        storage_location:
+          description: Storage root URL for model (for **MANAGED**, **EXTERNAL** models)
+          type: string
+        comment:
+          description: User-provided free-form text description.
+          type: string
+      required:
+        - model_name
+        - catalog_name
+        - schema_name
```

Comment:
nit: reorder, like above.

Reply:
Same comment as above.  We list them in this order in the documentation and protos.